lib/govuk_app_config/govuk_error/configure_defaults.rb

```diff
@@ -0,0 +1,68 @@
+module GovukError
+  class ConfigureDefaults < SimpleDelegator
```

Comment:
This doesn't need to be a simple delegator or have attributes as it is just calling methods on the config instance